tests/test_validate.py

```
@@ -0,0 +1,275 @@
+    status_code: int
+    payload: Any = field(default_factory=dict)
+
+    def json(self) -> Any:
+        return self.payload
+
+
+class _MockHttpClient:
+    def __init__(self, routes: dict[str, _MockResponse]) -> None:
+        self._routes = routes
+
+    def get(self, path: str, **kwargs: Any) -> _MockResponse:
```

Ruff (0.15.17)

[warning] Unused method argument: `kwargs`

(ARG002)